src/lib.rs

```diff
@@ -609,112 +720,4 @@ pub mod pallet {
         // Found duplicate of native function
         DuplicateNativeFunction,
     }
-
```

Comment:
Don't look at the red change block below, see more info [here](https://github.com/eigerco/pallet-move/pull/116/files#r1451510322)